config/boards/station-m2.wip

```diff
@@ -0,0 +1,12 @@
+# Rockchip RK3566
+BOARD_NAME="Station M2"
+BOARDFAMILY="rockchip64"
```

Comment:
Perhaps better to define BOARDFAMILY="station-p2"
then creating a file for family like https://github.com/armbian/build/blob/master/config/sources/families/rk3399.conf

What do you think?

Reply:
Maybe we could reuse rk35xx family?

Some of the variations (per soc) are already present there (like CPUMAX and blob sets) and kernel/u-boot sources could be varied the same way as they are in this PR.
Diverging sources within the family is not ideal but maybe we can unify them at some point similarly as it was done for P1/M1.

Reply:
> Maybe we could reuse rk35xx family?

Or that, yes.

Reply:
I agree that it is better to use a single 35xx structure, but .....  
I did it in this form, because I'm afraid that the new maintainer for rocpi3 has already started breaking this BOARDFAMILIE 35xx 

https://github.com/armbian/build/pull/3271

and I risk getting a non-working system and I will have to constantly fix it.  :)